vspace/src/main/webapp/WEB-INF/views/staff/dashboard/dashboard.html

```diff
@@ -46,8 +46,11 @@ <h4>Spaces</h4>
 			<h4>Modules</h4>
 			<div class="grid" style="margin-top: 10px;">
 				<div th:each="module,iterStat: ${recentModules}" th:if="${iterStat.index}<5" >
-                    <div class="btn-group">
-					<a th:href="@{'/staff/module/'+${module.id}}" class="btn btn-light" style="width: 10rem;">
+                    <div class="btn-group" >
+					<a th:href="@{'/staff/module/'+${module.id}}" class="btn btn-light" th:if="${module.name!=null && #strings.length(module.name)>20}" style="width:10rem;white-space: normal;overflow-wrap: break-word;">
```

Comment:
why don't you just always apply the style (overflow-wrap, white-space)? why only for module names longer than 20?